lib/commands/misc/giverole.js

```diff
@@ -26,14 +26,38 @@ module.exports = class GiveRoleCommand extends Command {
 
 	async run(msg, args) {
     if (args.role === "member") {
-      const role = msg.guild.roles.find(r => r.name == 'Member')
-      if (!msg.member.roles.has(role)) {
+      const role = msg.guild.roles.find(r => r.name == 'Member');
```

Comment:
It's best practice to always check with `===` because only a double `==` is a loose "abstract" comparison and may not give you the result you expect in some cases. Here's an [MDN of the difference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)